src/services/storageTypes.ts

```diff
@@ -0,0 +1,76 @@
+/*
+ * Copyright (c) 2018-2020 Red Hat, Inc.
+ * This program and the accompanying materials are made
+ * available under the terms of the Eclipse Public License 2.0
+ * which is available at https://www.eclipse.org/legal/epl-2.0/
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *
+ * Contributors:
+ *   Red Hat, Inc. - initial API and implementation
+ */
+
+export enum StorageTypeTitle {
+  async = 'Asynchronous',
+  ephemeral = 'Ephemeral',
+  persistent = 'Persistent',
+}
+
+export function toTitle(type: che.WorkspaceStorageType): string {
+  if (!StorageTypeTitle[type]) {
+    throw new Error(`Unknown storage type: "${type}"`);
+  }
+  return StorageTypeTitle[type];
+}
+
+export function fromTitle(title: string): che.WorkspaceStorageType {
+  switch (title) {
+    case StorageTypeTitle.async:
```

Comment:
Why don't you use StorageTypeTitle[StorageTypeTitle.async]?

Reply:
I'm going to do so.